source/features/previous-version.tsx

```diff
@@ -30,15 +31,16 @@ async function add(historyButton: HTMLElement): Promise<void> {
 	const url = new GitHubFileURL(location.href)
 		.assign({branch: previousCommit});
 
-	historyButton.before(
-		<a href={url.href} className="UnderlineNav-item tooltipped tooltipped-n ml-2" aria-label="View previous version">
-			<VersionsIcon className="UnderlineNav-octicon mr-0"/>
-		</a>,
-	);
+	const previousButton = historyButton.cloneNode(true);
+	previousButton.href = url.href;
+	$('span[data-component="leadingVisual"] svg', previousButton)
+		.replaceWith(<VersionsIcon className="UnderlineNav-octicon mr-0"/>);
```

Comment:
The class here needs to be dropped too. It might need some extra alignment style or classes

<img width="542" alt="Screenshot" src="https://github.com/refined-github/refined-github/assets/1402241/1fe12187-a5c4-4492-99a5-7f5c1f0a1b81">